lib/Sema/LookupVisibleDecls.cpp

```diff
@@ -415,7 +415,9 @@ static void lookupDeclsFromProtocolsBeingConformedTo(
     }
 
     DeclVisibilityKind ReasonForThisProtocol;
-    if (Reason == DeclVisibilityKind::MemberOfCurrentNominal)
+    if (Conformance->getKind() == ProtocolConformanceKind::Inherited)
+      ReasonForThisProtocol = getReasonForSuper(Reason);
```

Comment:
Could you explain why you added this?

Reply:
Ah, never mind. I got it.

Reply:
Since we now call the function once, we need to adjust the correct reason for an inherited conformance when the passed `Reason` is `MemberOfCurrentNominal`.

Reply:
(`LS->isOnSuperClass()` here is always false within `lookupDeclsFromProtocolsBeingConformedTo`)